src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Ball getRandomBall = Lottery.getRandomBall();
```

Comment:
```suggestion
            Ball randomBall = Lottery.getRandomBall();
```
The name of the variable should be a noun, not a verb